src/extension.ts

```diff
@@ -65,8 +66,8 @@ async function doActivate(_operationId: string, context: vscode.ExtensionContext
         });
     });
 
-    registerCommand(context, "maven.project.refreshAll", (): void => {
-        mavenExplorerProvider.refresh();
+    registerCommand(context, "maven.project.refreshAll", (item?: ITreeItem): void => {
```

Comment:
I've reused the command `refreshAll`, so it's now a "general" refresh action. Do you think I should change the command name? @jdneo

Reply:
I prefer `maven.explorer.refresh`